test/extended/authorization/authorization.go

```diff
@@ -1241,3 +1243,67 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] authorization", f
 		})
 	})
 })
+
+var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] authorization", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLI("bootstrap-policy")
+
+	g.Context("", func() {
+		g.Describe("PublicImageAccessWithBasicAuthShouldSucceed", func() {
+			g.It("should succeed [apigroup:image.openshift.io]", func() {
+				// Skip Hypershift external OIDC clusters
+				isExternalOIDCCluster, err := exutil.IsExternalOIDCCluster(oc)
+				o.Expect(err).NotTo(o.HaveOccurred())
+				if isExternalOIDCCluster {
+					g.Skip("Skipping the test as we are running against a Hypershift external OIDC cluster")
```

Comment:
Do we need this skip?

Reply:
In my test, I'm allowing unauthenticated users to access public images, but external OIDC clusters could interfere with this by enforcing authentication, which may block unauthenticated access and cause the test to fail for reasons unrelated to the functionality being tested. The g.Skip check is necessary to prevent running the test in such environments, ensuring it only runs where the authentication model aligns with the test's requirements. Without this skip, the test could become unreliable in external OIDC clusters, where authentication rules differ.

this is my understanding of the external OIDC clusters, if this is not the case please let me know.

Reply:
How does a cluster with an external OIDC IDP interfere with us testing the access for unauthenticated users?

Reply:
I mean it is not important to test this on an external OIDC cluster, if it is only HyperShift. But in a world, where most of our customers would use it,... it would be quite nice to test it there too. :)